Docs: Update patch for custom CA w ExtAuth (#5681)
solo-io/gloo: docs/content/installation/advanced_configuration/extauth_custom_ca.md

> To update an existing Gloo Edge Enterprise installation to support an additional trusted root certificate authority, we are going to patch the deployment for the external authentication server. You can do this by using `kubectl patch`. We are going to add three values for the volume, volumeMount, and initialization container.

Reviewer: Maybe shift the tense while we're editing this section?
```suggestion
To update an existing Gloo Edge Enterprise installation to support an additional trusted root certificate authority, you can patch the deployment for the external authentication server. As part of the patch, you add three values for the custom certificate authority: volume, volumeMount, and initialization container.
```

1. Get the current image of the external authentication pod.
   ```sh
   image=$(kubectl get deploy/extauth -n gloo-system -ojson | jq .spec.template.spec.containers[0].image -r)
   ```

> 2. Patch the `extauth` deployment with an initialization pod. When the patch is applied, the external authentication server pods are recreated. Note that the type of patch differs based on whether or not external authentication is running in TLS mode, in which the `envoy-sidecar` and the `sds` containers run in the external authentication pod.

Reviewer: ```suggestion
2. Patch the `extauth` deployment with an initialization container. When the patch is applied, the external authentication server pods are recreated. Note that the type of patch differs based on whether or not external authentication is running in TLS mode, in which the `envoy-sidecar` and the `sds` containers run in the external authentication pod.
```
